utils/utils.go

```diff
@@ -48,3 +52,113 @@ func RemoveDuplicatesFromStringArray(arr []string) []string {
 
 	return result
 }
+
+// OutputJSONHeader holds the received Header to test sending emails (inbucket)
+type JSONMessageHeaderInbucket []struct {
```

Comment:
Can we move all the stuff you added to this file into a new file inside `platform/utils/`? I think it would make sense for it to be contained in it's own file within the `utils` package. Maybe call it `inbucket.go` or something like that